src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonStringEnumConverter.cs

```diff
@@ -6,6 +6,51 @@
 
 namespace System.Text.Json.Serialization
 {
+    public class JsonStringEnumConverter<TEnum> : JsonConverterFactory
```

Comment:
XML doc comments documenting the type?

Reply:
Also, should it be sealed?

Reply:
> XML doc comments documenting the type?

I'm actually surprised that this ended up compiling.

Reply:
> Also, should it be sealed?

We want users to be able to derive from the class so that they can specify their own naming policy.